Maker-Agents-Suit-Module/Governance-Module/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "maker-gm",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
Not sure if we should move to version 0.0.2 while the first one wasn't even deployed